shared/substrate/events.go

```diff
@@ -186,34 +119,25 @@ type EventMultisigCancelled struct {
 
 type Events struct {
 	types.EventRecords
-	ChainBridge_RelayerThresholdChanged []EventRelayerThresholdChanged        //nolint:stylecheck,golint
-	ChainBridge_ChainWhitelisted        []EventChainWhitelisted               //nolint:stylecheck,golint
-	ChainBridge_RelayerAdded            []EventRelayerAdded                   //nolint:stylecheck,golint
```

Comment:
why did we remove the Relayer related & Proposal related events from here? Are they duplicated elsewhere? If so would it make sense to have all events here?

Reply:
All chainbridge events are now located here: https://github.com/ChainSafe/chainbridge-substrate-events/blob/master/events.go

Part of the ticket was to reference the new package rather than internally.